package.json

```
@@ -3,9 +3,13 @@
   "version": "2.7.0",
   "description": "A graphql-subscriptions PubSub Engine using redis",
   "main": "dist/index.js",
+  "publishConfig": {
+    "registry": "https://lvt.jfrog.io/artifactory/api/npm/npm/",
+    "directory": "build"
```

```suggestion
    "directory": "dist"
```
I believe this is supposed to be the path to the outputted distribution. looks like they write things out to `dist` instead of `build` like the example I sent you